### PR TITLE
fix: use design guideline color

### PR DESCRIPTION
**Pull request by Aylie-Chou**
# Issue
[[維運] 網站顏色盤點 : 主網站](https://app.asana.com/0/1203699264568808/1206157706155674/f)

# Notice
ref: [twreporter color](https://docs.google.com/spreadsheets/d/1-g8lEFgB4tbvXIunl0xOLfjsdlCghjohuvfiHZP0BGA/edit?usp=sharing)

# Dependency
https://github.com/twreporter/twreporter-npm-packages/pull/504

### CODE REVIEW

File: src/components/photography/article-list.js [hunk 1]
```diff
@@ -136,14 +140,14 @@ const List = styled.ul`
   }
 
   ${ItemTitle}, ${ItemExcerpt}, ${Date} {
-    color: #F7F7F7;
+    color: ${colorGrayscale.gray100};
   }
   ${Item}:hover {
     ${ItemTitle}, ${Date} {
       color: black;
```
Comment (liruchen32) [inline src/components/photography/article-list.js:147]: Do we need to change `black` to `colorGrayscale.black`

File: src/components/photography/top-news.js [hunk 2]
```diff
@@ -51,7 +52,7 @@ const CategoryContainer = styled.div`
 
 const Card = styled.div`
   position: absolute;
-  background-color: rgba(255, 255, 255, 0.9);
+  background-color: ${colorOpacity['white_0.9']};
   border: 2px solid black;
```
Comment (liruchen32) [inline src/components/photography/top-news.js:56]: Do we need to change `black` to `colorGrayscale.black`

**Review by liruchen32 — CHANGES_REQUESTED** [review]
Great efforts for changing color to system color. Just a few little questions as comments.

File: src/components/photography/article-list.js (hunk 1 above)
Comment (Aylie-Chou) [reply src/components/photography/article-list.js]: fixed ([d30b42](https://github.com/twreporter/twreporter-react/pull/2772/commits/d30b42cd3cca96f616a7bdaf6e20d94e6c523100))

File: src/components/photography/top-news.js (hunk 2 above)
Comment (Aylie-Chou) [reply src/components/photography/top-news.js]: fixed ([d30b42](https://github.com/twreporter/twreporter-react/pull/2772/commits/d30b42cd3cca96f616a7bdaf6e20d94e6c523100))

**Review by liruchen32 — APPROVED** [review]
LGTM

**Pull request merged**